extensions/amp-selector/0.1/amp-selector.js

```diff
@@ -292,9 +292,8 @@ export class AmpSelector extends AMP.BaseElement {
               targetOption: el.getAttribute('option'),
               selectedOptions: selectedValues,
             });
-        // TODO(choumx, #9699): HIGH.
         this.action_.trigger(this.element, name, selectEvent,
-            ActionTrust.MEDIUM);
+            ActionTrust.HIGH);
```

Comment:
It wouldn't since high trust events can do everything that medium trust events can.